Gopkg.toml

```diff
@@ -208,7 +208,7 @@ required = [
 # more easily.)
 [[constraint]]
   name = "github.com/open-policy-agent/opa"
-  version = "=0.10.7"
+  revision = "7864d60dd78d748dbce54b569e939f5b0dc07486"
```

Comment:
That's the OPA code change allowing for the disable-inline-stuff. We can go with 0.13.0 when that release is cut.